ServerCore/Pages/Pieces/InternMeta.cshtml.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using ServerCore.DataModel;
+using ServerCore.Helpers;
+using ServerCore.ModelBases;
+
+namespace ServerCore.Pages.Pieces
+{
+    [Authorize(Policy = "PlayerIsOnTeam")]
```

Comment:
During PH20, I would have appreciated, as an author, a way to see the piece list (or at least piece count) for all teams or for a specific team.  So, as a bridge to that future, you might want to let the puzzle author view this page, not just members of the team.  (But presumably not in this initial no-frills version!)

Reply:
Not in the no-frills version but I understand the desire. We might implement general impersonation to address this.